stdlib/public/SwiftShims/MetadataSections.h

```diff
@@ -0,0 +1,54 @@
+//===--- MetadataSections.h --------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+
```

Comment:
Can you get rid of one line here and add a doxygen file level comment:

https://llvm.org/docs/CodingStandards.html#file-headers

(Modulo of course the LLVM part of the header).